### DISCUSSION TITLE
Material theme version

### DISCUSSION THREAD

**Discussion by hhoeflin** [mkdocs/mkdocs#2965] [General]
Hi, 

I hope this is not a duplicate but i could not find the information. I was wondering which version of the material components are being used and if all of them are packaged by default or only a subset.

Thanks

**Comment by fu-sen** [comment]
hmm? I think it refers to [mkdocs-material](https://github.com/squidfunk/mkdocs-material). It's a separate project from MkDocs, so it's probably not something I'd get a comment on here.

**Reply by hhoeflin** [reply]
Oh yes of course. Thanks
